src/tls13.c

```diff
@@ -8458,13 +8459,23 @@ static int SendTls13Certificate(WOLFSSL* ssl)
 {
     int    ret = 0;
     word32 certSz, certChainSz, headerSz, listSz, payloadSz;
-    word16 extSz = 0;
+#if defined(HAVE_CERTIFICATE_STATUS_REQUEST)
+    TLSX* ext;
+    CertificateStatusRequest* csr;
+    #define MAX_CERT_EXTENSIONS MAX_CHAIN_DEPTH
+#else
+    #define MAX_CERT_EXTENSIONS 1
+#endif
+    word16 extSz[1 + MAX_CERT_EXTENSIONS] = { OPAQUE16_LEN };
```

Comment:
extSz is memset later ... is the initializer needed?

Reply:
Don't need. Removed.